war/package.json

```
@@ -14,8 +14,8 @@
     "prod": "webpack --config webpack.config.js --mode=production",
     "build": "yarn prod",
     "start": "yarn dev --watch",
-    "lint:js": "eslint . --ext js",
-    "lint:js-ci": "eslint . --ext js -f checkstyle -o target/eslint-warnings.xml",
+    "lint:js": "eslint ../ --ext js",
+    "lint:js-ci": "eslint ../ --ext js -f checkstyle -o target/eslint-warnings.xml",
```

```suggestion
    "lint:js-ci": "eslint ../ --ext js -f checkstyle -o ../target/eslint-warnings.xml",
```
Let's move the output one directory up too.

---

I think it’s fine where it is, it’s current working directory is the war folder

---

The package.json is, yeah, but ESLint now runs on all directories, you'd expect to find its results in target/eslint-warnings over war/target/eslint-warnings, no? 🤔

---

Is this enough to hold back an approval?

I’d expect to find it wherever it says it puts it, either work

---

> Is this enough to hold back an approval?

No 😎